_sponsors/inria.md

```diff
@@ -0,0 +1,6 @@
+---
+name: "BitFury"
+link: "https://bitfury.com/"
```

Comment:
This one looks wrong. The filename says inria, but it looks like a copy of bitfury.md

Reply:
Yup this one is wrong. :)